src/utils/__tests__/media-utils.spec.ts

```diff
@@ -0,0 +1,149 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import {
+  MEDIA_DIRECTORY_NAME,
+  MEDIA_FILE_NAME,
+  MEDIA_FILE_SHA,
+  MEDIA_SITE_NAME,
+  dirInput,
+  imageFilePrivateInput,
+  imageFilePublicInput,
+  mediaDir,
+  pdfFilePrivateInput,
+  pdfFilePublicInput,
+  svgFilePrivateInput,
+  svgFilePublicInput,
+} from "@root/fixtures/media"
+
+import { getMediaFileInfo, isMediaFileOutput } from "../media-utils"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+
+jest.mock("@utils/token-retrieval-utils", () => ({
+  getAccessToken: jest.fn().mockResolvedValue("token"),
+}))
+jest.mock("axios", () => ({
+  get: jest.fn().mockResolvedValue({
+    data: "blahblah",
+    headers: {
+      "content-type": "blahblah",
+    },
+  }),
+}))
```

Comment:
style: we alr have `mock-axios` btw so this is done automatically and we should prefer using that instead